externtype.go

```diff
@@ -46,8 +46,8 @@ func (ty *ExternType) FuncType() *FuncType {
 	return mkFuncType(ptr, ty.owner())
 }
 
-// GlobalType returns the underlying `GlobalType` for this `ExternType` if it's a function
-// type. Otherwise returns `nil`.
+// GlobalType returns the underlying `GlobalType` for this `ExternType` if it's a *global* type.
```

Comment:
Sorry for the confusion! When I said *global* type I put it in asterisks to call out that's the word that has to change, not to use the literal asterisks.

This function should have `..if it's a global type.`

`TableType` should have `...if it's a table type.`

`MemoryType` should have `...if it's a memory type.`

Reply:
Sorry, my fault